README.md

```diff
@@ -257,6 +257,49 @@ codacy-analysis-cli analyse \
 
 > In alternative to setting `--api-token` you can define CODACY_API_TOKEN in the environment.
 
+## GitHub Action
+
+The analysis CLI is available to be executed as a GitHub Action. The following is an example of a workflow
+using the CLI as an action, to analyse each commit and pull request, with your Codacy configuration.
+
+```yaml
+name: codacy-analysis-cli
+
+on: ["push"]
+
+jobs:
+  codacy-analysis-cli:
+    runs-on: ubuntu-latest
+    name: codacy-analysis-cli
+    steps:
+      - uses: actions/checkout@master
+      - name: Run codacy-analysis-cli
+        uses: codacy/codacy-analysis-cli@master
+        with:
+          project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+```
+
+Running the action with the default configurations will:
+
+- Analyse the current commit or pull request, using your Codacy configurations, running all supported
+    tools for the languages you are using.
+- Print analysis results into the console (you can check them in GitHub's action workflow panel).
+- Fail the workflow if at least one issue is found in your code.
+
+Check the next section to see how you can further configure this action.
+
+### Caveats
+
+This action supports all [CLI configuration options](#commands-and-configuration) with the following exceptions:
+
+- `--codacy-api-base-url` -- **Not supported**. The action will only get your configurations from Codacy cloud.
```

Comment:
I was thinking about supporting this just for Cloud users, but for no real reason. I think we can support enterprise as well. Of course that it will only be available to the enterprise users that use GitHub cloud.

Reply:
I think we could do it